src/Controller/MyController.php

```diff
@@ -11,7 +11,7 @@ class MyController extends AbstractController
 {
     /**
      * @RouteScope(scopes={"sales-channel-api"})
-     * @Route("/sales-channel-api/v1/swag/my-sales-channel-api-action", name="sales-channel-api.action.swag.my-sales-channel-api-action", methods={"GET"})
+     * @Route("/sales-channel-api/v2/swag/my-sales-channel-api-action", name="sales-channel-api.action.swag.my-sales-channel-api-action", methods={"GET"})
```

Comment:
please do not hardcode the version, use `v{version}` instead. have a look at the core or other plugins for examples :blush: